### PR TITLE
Add Zenodo DOIs for example data

### PR DESCRIPTION
**Pull request by rpauszek**
**Why this PR?**
So users can have example data to try out the tutorials!

[docs build here](https://lumicks-pylake.readthedocs.io/en/zenodo_dois/tutorial/index.html)

### CODE REVIEW

**Review by JoepVanlier — APPROVED** [review]
LGTM! Just give it a quick test and :shipit:

File: docs/tutorial/file.rst [hunk 1]
```diff
@@ -128,26 +128,34 @@ All channel data can be accessed using the above method. High frequency force da
...
+    plt.show()
 
 The `timestamps` attribute returns the measurement time in nanoseconds since epoch (January 1st 1970, midnight UTC/GMT). Note that since these values are typically very large, they cannot be converted to floating point without losing precision::
 
-    >>>t = f1x_timestamps[0]
-    >>>roundtrip_t = np.int64(np.float64(t))
-    >>>print(t - roundtrip_t)
+    t = f1x_timestamps[0]
+    roundtrip_t = np.int64(np.float64(t))
+    print(t - roundtrip_t)
```
Comment (JoepVanlier) [inline docs/tutorial/file.rst:149]: >>>

File: lumicks/pylake/__about__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 __title__ = "lumicks.pylake"
-__version__ = "0.13.3"
+__version__ = "1.0.0"
```
Comment (JoepVanlier) [inline lumicks/pylake/__about__.py:2]: 200w

**Comment by rpauszek** [comment]
pie

**Pull request merged**